utils/lib/Quickstart.ts

```diff
@@ -0,0 +1,269 @@
+import * as fs from 'fs';
+import * as yaml from 'js-yaml';
+import * as path from 'path';
+import * as glob from 'glob';
+
+import Alert from './Alert';
+import Dashboard from './Dashboard';
+import DataSource from './DataSource';
+import {
+  MOCK_UUID,
+  GITHUB_RAW_BASE_URL,
+  QUICKSTART_MUTATION,
+} from '../constants';
+import { getAssetSourceUrl, removeBasePath } from './classHelpers';
+import {
+  fetchNRGraphqlResults,
+  getCategoryTermsFromKeywords,
+} from './nr-graphql-helpers';
+import type {
+  QuickstartMutationVariable,
+  QuickstartMetaData,
+  QuickstartSupportLevel,
+} from '../types/QuickstartMutationVariable';
+import type { QuickstartConfig } from '../types/QuickstartConfig';
+import InstallPlan from './InstallPlan';
+
+interface QuickstartMutationResponse {
+  quickstart: {
+    id: string;
+  };
+}
+
+interface SupportLevelMap {
+  [key: string]: QuickstartSupportLevel;
+}
+
+const SUPPORT_LEVEL_ENUMS: SupportLevelMap = {
+  'New Relic': 'NEW_RELIC',
+  Community: 'COMMUNITY',
+  Verified: 'VERIFIED',
+};
+
+type ComponentType =
+  | typeof Alert
+  | typeof Dashboard
+  | typeof DataSource
+  | typeof InstallPlan;
```

Comment:
Since install plans are not being pull into this class, let's remove it.
```suggestion
  | typeof DataSource;
```